source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+     - none of the values specified in an array.
+
+   * - :java-core-api:`empty() <com/mongodb/client/model/Filters.html#empty()>`
+     - all the documents.
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field equals "5" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin equalComparison
+   :end-before: end equalComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$eq": { "qty": 5 } }
+
+The output of the above code snippet should look something like this:
+
+.. code-block:: json
+   :copyable: false
+   
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+The following example creates a filter that matches all documents where
+the value of the ``qty`` field is greater than or equal to "8" in the
+``paint_purchases`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin gteComparison
+   :end-before: end gteComparison
+   
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$gte": { "qty": 8 } }
+
+The output of the above code snippet should look something like this:
+
+.. code-block:: json
+   :copyable: false
+   
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 7, "color": "green", "qty": 8, "vendor": ["B", "C"]}
+
+The following example creates a filter that matches all documents in
+the ``paint_purchases`` collection because the predicate is empty: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin emptyComparison
+   :end-before: end emptyComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { }
+
+The output of the above code snippet should look something like this:
+   
+.. code-block:: json
+   :copyable: false
+      
+   {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+   {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+   {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+   {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+   {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+   {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+   {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+   {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+.. _logical:
+
+Logical
+-------
+The logical operator looks at the conditions of the specified filters.
```

Comment:
Can you elaborate on this? I'm not sure if I could understand how they function from the description: "looks at the conditions".